qiskit_addon_sqd/fermion.py

```diff
@@ -16,14 +16,21 @@
 from __future__ import annotations
 
 from dataclasses import dataclass
+from typing import Callable, cast
 
 import numpy as np
 from jax import Array, config, grad, jit, vmap
 from jax import numpy as jnp
 from jax.scipy.linalg import expm
 from pyscf import fci
+from pyscf.fci.selected_ci import _as_SCIvector, make_rdm1, make_rdm1s, make_rdm2, make_rdm2s
```

Comment:
Should we make an issue over at pyscf to add something to the API to fill in for `_as_SCIvector`?